### PR TITLE
Allow for automatic subscription to be read from subscription policy configs

### PR DESCRIPTION
**Pull request by zturechek**
Immuta application v2020.3.2 now allows for automatic subscription to data sources to be specified in subscription policies. This means users are not required to take manual action within the UI to subscribe individually or in bulk to data sources if they are allowed to by the policy.

Changes here allow for reading this boolean from subscription policy configs. Tested on v2020.3.2.

### CODE REVIEW

File: fh_immuta_utils/policy.py [hunk 1]
```diff
@@ -144,6 +144,7 @@ class SubscriptionPolicyAction(BaseModel):
     subscriptionType: str
     exceptions: Optional[PolicyExceptions] = None
     allowDiscovery: bool = False
+    automaticSubscription: bool = True
```
Comment (chintanms) [inline fh_immuta_utils/policy.py:147]: For backward compatibility, should default be False?

File: fh_immuta_utils/policy.py [hunk 2]
```diff
@@ -355,6 +360,9 @@ def make_global_subscription_policy(
             action = make_subscription_policy_action(
                 exceptions_config=action_grouping["exceptions"],
                 allow_discovery=action_grouping.get("allowDiscovery", False),
+                automatic_subscription=action_grouping.get(
+                    "automaticSubscription", True
```
Comment (chintanms) [inline fh_immuta_utils/policy.py:364]: Same here, should default be False?

**Review by chintanms — COMMENTED** [review]
Couple of comments, rest of the changes look good to me.

File: fh_immuta_utils/policy.py (hunk 1 above)
Comment (zturechek) [reply fh_immuta_utils/policy.py:147]: The default in Immuta is True. Tested by enrolling a policy without this parameter specified in the configs. You can also see it in the UI when creating a new subscription policy.

**Review by chintanms — APPROVED** [review]
lgtm!

**Comment by zturechek** [comment]
Reverted `CHANGELOG.md` changes to utilize the new pattern of release notes introduced in [https://github.com/flatironhealth/fh-immuta-utils/pull/13](https://github.com/flatironhealth/fh-immuta-utils/pull/13)

**Pull request merged**